README.md

```diff
@@ -140,4 +140,4 @@ All types of contributions are encouraged and valued. See the [Contributing](CON
 
 ## 📘 License
 
-This project is released under the under terms of the [ISC License](LICENSE).
+This project is released under the under terms of the [ISC License](LICENSE).
```

Comment:
What's the difference here? Maybe LF?

Reply:
Yes, i managed to reverte it by overwriting it with the original source.